spec/models/alert_spec.rb

```diff
@@ -24,6 +25,8 @@
   let(:course) { create(:course) }
   let(:revision) { create(:revision) }
   let(:user) { create(:user) }
+  let(:alert) { create(:alert) }
```

Comment:
one confusing thing here is that the test of `.resolvable?` below is implicitly relying on the default 'alert' factory to return an ArticlesForDeletionAlert. You should specify the type and the `resolved` value explicitly here, so that it's more clear exactly what you're testing.